src/ProtocolV3TestBase.sol

```diff
@@ -307,6 +318,26 @@ contract ProtocolV3TestBase is CommonTestBase {
       / 100_00;
   }
 
+  function _isAboveBorrowCap(
+    IPool pool,
+    ReserveConfig memory borrowConfig,
+    uint256 borrowAmount
+  ) internal view returns (bool) {
+    if(keccak256(abi.encodePacked(borrowConfig.symbol)) == keccak256(abi.encodePacked('DAI')))
+      return false;
```

Comment:
Why is there an exception here? Ideally we want the code to be as generic as possible across multiple instances.